Detectors/TPC/reconstruction/macro/createTPCSpaceChargeCorrection.C

```diff
@@ -81,7 +81,7 @@ void createTPCSpaceChargeCorrection(
   std::unique_ptr<TPCFastTransform> fastTransform(TPCFastTransformHelperO2::instance()->create(0));
 
   std::fstream file(outputFileName, std::fstream::in | std::fstream::out);
```

Comment:
``fastTransform->writeToFile`` will open a root file itself, why do you need to open a generic fstream here? Looks like a bug.

Reply:
Hello @shahor02 ,
indeed that was a bug. Fixed now.